### ISSUE TITLE
is this good for chat system? 

### ISSUE THREAD

**Issue by Dj-jom2x** [rook2pawn/node-ddos#17]
i mean you now  its a chat it floods ..  example 10 user same ip . they chat .. does it counted ?

**Comment by Dj-jom2x** [comment]
ddos_1.default is not a constructor

**Comment by rook2pawn** [comment]
@Dj-jom2x Yes, it would be quite good for a chat system. However, the user who is flooding the system will need some kind of feedback on how long they have to wait. It's a good idea to provide that kind of information as a configurable option. Keeping this thread open as its a very good feature.
